setup.py

```diff
@@ -543,6 +543,7 @@ def run(self):
 include_dirs += [
     cwd,
     os.path.join(cwd, "torch", "csrc"),
+    os.path.join(cwd, "torch", "csrc", "api", "include"),
```

Comment:
My idea was users will want to just point their compiler at `torch/csrc/api/include`, and then use e.g. `#include <torch/nn/module.h>`, instead of the full path.